fix(package-json): invalid webpack commit sha

**Describe the bug**

When trying to set up the project from  scratch (such as the CI is doing it)

you get the error

```sh
Run npm ci
  npm ci
  shell: /usr/bin/bash -e {0}
  env:
    JAVA_HOME_8.0.275_x64: /opt/hostedtoolcache/jdk/8.0.275/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.275/x64
    JAVA_HOME_8_0_275_X64: /opt/hostedtoolcache/jdk/8.0.275/x64
npm ERR! code 128
npm ERR! Command failed: git checkout a1021fd674dd24b13d759611b77b5d7057f7f0ea
npm ERR! fatal: reference is not a tree: a1021fd674dd24b13d759611b77b5d7057f7f0ea
npm ERR! 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-01-21T02_12_54_047Z-debug.log


```

**To Reproduce**

```sh
cd $PROJECT_ROOT
rm package-lock.json
rm -rf node_modules
npm install
```

**Expected behavior**

npm install should not fail

**Logs/Stack traces**

```sh
Run actions/setup-node@v2.1.2
  with:
    node-version: v12.13.0
    always-auth: false
    check-latest: false
    token: ***
  env:
    JAVA_HOME_8.0.275_x64: /opt/hostedtoolcache/jdk/8.0.275/x64
    JAVA_HOME: /opt/hostedtoolcache/jdk/8.0.275/x64
    JAVA_HOME_8_0_275_X64: /opt/hostedtoolcache/jdk/8.0.275/x64
Attempting to download v12.13.0...
Acquiring 12.13.0 from https://github.com/actions/node-versions/releases/download/12.13.0-20200507.88/node-12.13.0-linux-x64.tar.gz
Extracting ...
/usr/bin/tar xz --strip 1 --warning=no-unknown-keyword -C /home/runner/work/_temp/2a9375d2-0ba6-4926-9c77-36a30f5fc64d -f /home/runner/work/_temp/bc132599-8217-4c6f-8c16-5e8ec879caa5
Adding to the cache ...
Done

```

**Operating system name, version, build:**

Affects all versions of all OS'

**Hyperledger Cactus release version or commit (git rev-parse --short HEAD):**

`main`

**Hyperledger Cactus Plugins/Connectors Used**

`N/A`

**Additional context**

The git commit SHA in the dependency version for webpack is necessary because there's been an open PR for webpack (to support a feature we need) for quite some time and it is unknown when if ever it will get merged unfortunately.

cc: @sfuji822 @takeutak @hartm @jonathan-m-hamilton

--- a/package.json
+++ b/package.json
@@ -93,7 +93,7 @@
     "ts-node": "8.9.1",
     "tslint": "6.0.0",
     "typescript": "4.0.3",
-    "webpack": "git+https://github.com/petermetz/webpack.git#a1021fd674dd24b13d759611b77b5d7057f7f0ea",
+    "webpack": "git+https://github.com/petermetz/webpack.git#cactus-webpack-ignore-require-calls-feature",
     "webpack-bundle-analyzer": "3.6.0",
     "webpack-cli": "3.3.11"
   },

